examples/debian_snapshot/BUILD.bazel

```diff
@@ -54,66 +54,52 @@ cacerts(
     }),
 )
 
-PACKAGES = [
-    "@bullseye//ncurses-base",
-    "@bullseye//libncurses6",
-    "@bullseye//tzdata",
-    "@bullseye_nolock//bash",
-    "@bullseye//coreutils",
-    "@bullseye//dpkg",
-    "@bullseye//apt",
-    "@bullseye//perl",
-    "@bullseye//openssl",
-    "@bullseye//nvidia-kernel-common",
-]
-
-# Creates /var/lib/dpkg/status with installed package information.
-dpkg_status(
-    name = "dpkg_status",
-    controls = select({
-        "@platforms//cpu:arm64": [
-            "%s/arm64:control" % package
-            for package in PACKAGES
-        ],
-        "@platforms//cpu:x86_64": [
-            "%s/amd64:control" % package
-            for package in PACKAGES
-        ],
-    }),
-)
-
 oci_image(
-    name = "apt",
-    architecture = select({
-        "@platforms//cpu:arm64": "arm64",
-        "@platforms//cpu:x86_64": "amd64",
-    }),
+    name = "image",
+    architecture = "arm64",
     env = {
         # Required to use the SSL certs from `cacerts()`
         "SSL_CERT_FILE": "/etc/ssl/certs/ca-certificates.crt",
     },
     os = "linux",
     tars = [
+        # This target contains all the installed packages.
+        "@bullseye//:bullseye",
```

Comment:
just a nit, but I guess the packages should go at the end of the `tars` so that if they change, the previous layers remain the same.